src/lib/y2storage/autoinst_proposal.rb

```diff
@@ -19,42 +19,29 @@
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
 
-require "yast"
-require "y2storage/storage_manager"
-require "y2storage/disk_analyzer"
-require "y2storage/proposal"
+require "y2storage/proposal_settings"
```

Comment:
Why? The `ProposalSettings` are only useful for `GuidedProposal`. Why was this require added?